wallet/wallet-controller/src/synced_controller.rs

```diff
@@ -580,6 +581,32 @@ impl<'a, T: NodeInterface, W: WalletEvents> SyncedController<'a, T, W> {
         .await
     }
 
+    pub async fn decommission_stake_pool_request(
+        &mut self,
+        pool_id: PoolId,
+    ) -> Result<HexEncoded<PartiallySignedTransaction>, ControllerError<T>> {
```

Comment:
The wallet controller should not return HexEncoded stuff. HexEncoding should happen at the UI level. Not every system that uses the controller needs hex encoding, this is a UI thing.